### DISCUSSION TITLE
it takes too long to pull streamers

### DISCUSSION THREAD

**Discussion by Natsu702** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#370] [Q&A]
`24/12/21 08:05:50 - ERROR - [__init__]: Unable to detect if you have the latest version of this script

24/12/21 08:05:50 - INFO - [run]: Start session: '3850ea62-471d-4670-966d-119e19bc8c0b'

24/12/21 08:05:50 - INFO - [run]: Loading data for 11 streamers. Please wait...

24/12/21 08:05:58 - INFO - [run]: Streamer streamer-username06 does not exist

24/12/21 08:05:59 - INFO - [run]: Streamer streamer-username07 does not exist

24/12/21 08:06:04 - INFO - [run]: Streamer streamer-username08 does not exist

24/12/21 08:06:05 - INFO - [run]: Streamer streamer-username09 does not exist

24/12/21 08:06:07 - INFO - [run]: Streamer streamer-username10 does not exist

24/12/21 08:06:08 - INFO - [run]: Streamer streamer-username11 does not exist

24/12/21 08:09:00 - INFO - [set_offline]: Streamer(username=StreamerUsername, channel_id=123, channel_points=15.15k) is Offline!

24/12/21 08:13:18 - INFO - [set_offline]: Streamer(username=streamerUsername, channel_id=123, channel_points=9.33k) is Offline!
`

**Comment by Tkd-Alex** [comment]
You should remove the place holder streamer-username 1/2/3 etc
